python/lsst/dax/imgserv/cli.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
```

Comment:
Just for convenience, as I've seen it done elsewhere in LSST.  But then it's a py file, so something like cli* would have made more sense.  But let's leave it as is for now.